File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int i=2,j=2,n=nums.size();
        while(i<n){
            if(nums[i]!=nums[j-2]){
                nums[j]=nums[i];
                j++;
            }
            i++;
        }
        return min(n,j);
    }
};

File: 0080-remove-duplicates-from-sorted-array-ii/0080-remove-duplicates-from-sorted-array-ii.py
class Solution:
    def removeDuplicates(self, nums: List[int]) -> int:
        j = 2
        n = len(nums)
        for i in range(2, n):
            if nums[i] != nums[j - 2]:
                nums[j] = nums[i]
                j += 1
        return j